core/src/main/java/com/linecorp/armeria/server/GracefulShutdownSupport.java

```diff
@@ -56,6 +56,11 @@ static GracefulShutdownSupport disabled() {
      */
     public abstract void dec();
 
+    /**
+     * Returns the number of pending requests.
```

Comment:
nit: `requests` -> `responses` for consistency?

Reply:
Oops, fixed